qiita_db/study.py

```diff
@@ -340,6 +360,17 @@ def create(cls, owner, title, efo, info, investigation=None):
                "efo_id) VALUES (%s, %s)".format(cls._table))
         conn_handler.executemany(sql, [(study_id, e) for e in efo])
 
+        # insert portal into database
+        portal_id = convert_to_id(qiita_config.portal, 'portal_type', 'portal')
+        portal_sql = ""
+        if qiita_config.portal != "QIITA":
```

Comment:
It allows for very quick grabbing of all studies in the portal, and allows us to have a universal way of getting studies by just asking for studies by portal. Since it's a one-to-many anyway, this ease of use is worth it. Every study belongs to QIITA as well as whatever other portal.